js/index.js

```diff
@@ -1,144 +1,252 @@
-			{ name: "Thermal and Fluid Analysis", rating: 3 },
-			{ name: "Bluetooth Low-Energy", rating: 2 },
-		]
-	},{
-		title: "Non-Technical",
-		entries: [
-			{ name: "UI Design", rating: 3 },
-			{ name: "UX Research", rating: 3 },
-			{ name: "Project Management", rating: 2},
-		]
-	},
+const SKILLS = [
+  {
+    title: "Languages",
+    entries: [
+      { name: "JavaScript", rating: 5 },
+      { name: "Python", rating: 3 },
+      { name: "Java/Scala", rating: 2 },
+      { name: "Ruby", rating: 2 },
+      { name: "C/C++", rating: 2 },
+    ],
+  },
+  {
+    title: "Architecture",
+    entries: [
+      { name: "Relational Databases", rating: 4 },
+      { name: "ETL Pipelines", rating: 3 },
+      { name: "Google Cloud / AWS", rating: 3 },
+      { name: "Redis & RabbitMQ", rating: 2 },
+      { name: "Docker/Kubernetes", rating: 2 },
+    ],
+  },
+  {
+    title: "Non-Engineering Skills",
+    entries: [
+      { name: "UI/UX Design", rating: 3 },
+      { name: "Project Management", rating: 2 },
+      { name: "A/B Experiment Design", rating: 2 },
+      { name: "Copy Writing", rating: 2 },
+    ],
+  },
 ];
 
+const EXPERIENCE = [
+  {
+    title: "Loom, Inc.",
+    link: "https://loom.com",
+    metadata: ["Senior Software Engineer", "September 2021 - June 2022"],
+    tasks: [
+      "Led feature development for a high-traffic Slack integration using queues and rate-limiting",
+      "Collaborated with data analysts, UI designers, and product managers to drastically expand product presence for a chrome extension with >100,000 daily active users",
+      "Conducted A/B experiments to test our assumptions on user behavior and make more informed product decisions",
+    ],
+  },
 
-function initSkills(container){
-	for( var i in skillSets ){
-		var skillSet = skillSets[i];
+  {
+    title: "Wakebird",
+    link: "https://wakebird.com",
+    metadata: ["Co-Founder", "January 2019 - Present"],
+    tasks: [
+      "Led a two-person team to design, build, and sell a web application for organizing dragon boat racing teams",
+      "Mentored a junior engineer on code review, sprint planning, node.js, and React",
+      "Synthesized interviews with athletes and coaches into meaningful feature design specs",
+      "Taught athletes and coaches how to use Wakebird, and solicited feedback to get to the bottom of customer needs",
+      "Reviewed bookkeping ledger and tax remittance on a quarterly basis",
+    ],
+  },
 
-		var columnDiv = $("<div/>",{
-			"class":"column"
-		}).appendTo(container);
+  {
+    title: "Flatiron Health",
+    link: "https://flatiron.com",
+    metadata: ["Senior Software Engineer", "June 2019 - August 2021"],
+    tasks: [
+      "Drafted and built the infrastructure for a machine-learning-based product that predicts patient ER visits for oncology nurses",
+      "Refactored and maintained complex Extract, Transform & Load (ETL) pipelines with Apache Airflow for quarterly delivery of abstracted cancer data",
```

Comment:
"refactor" is a red flag for experienced engineers reading this. Not because refactoring is bad but because people waste company time re-arranging code. So you don't want to come off as one of those devs, who moves code just because it makes better sense to them.

Refactors should always be associated with a real problem with the code base and outcome as a result of cleaning tech debt? Did the refactor make it a lot easier for other engineers to work on the code-base? Did the refactor lower the frequency of bugs?
Place "refactor and maintained" with more active and impactful verbs.

Reply:
Wait... is this a thing? Refactoring isn't frowned upon anywhere I've been. I've only seen one engineer that did it egregiously (like made it almost 100% of the time he spent).